src/helpers/handle-resuming.ts

```diff
@@ -0,0 +1,98 @@
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import { LavalinkManager, NodeManager } from 'lavalink-client';
+import { PlayerManager } from './player-manager';
+import { Client } from 'discord.js';
+import { LAVALINK_MODULE_OPTIONS } from '../necord-lavalink.module-definition';
+import { NecordLavalinkModuleOptions } from '../necord-lavalink-options.interface';
+import { PlayerStore } from '../constants';
+import { BaseStore } from './base-store';
+import { PlayerSaver } from './player-saver';
+
+@Injectable()
+export class ResumingHandler {
+	public constructor(
+		private readonly nodeManager: NodeManager,
+		private readonly playerManager: PlayerManager,
+		private readonly lavalinkManager: LavalinkManager,
+		private readonly client: Client,
+		private readonly playerSaver: PlayerSaver,
+		@Inject(PlayerStore)
+		private readonly store: BaseStore,
+		@Inject(LAVALINK_MODULE_OPTIONS)
+		private readonly lavalinkOptions: NecordLavalinkModuleOptions
+	) {}
+
+	private readonly logger = new Logger('ResumingHandler');
```

Comment:
```suggestion
	private readonly logger = new Logger(ResumingHandler.name);
```

Also all properties must be on top of constructor